components/dashboard/src/admin/Settings.tsx

```diff
@@ -5,33 +5,45 @@
  */
 
 import { useContext } from "react";
-import { InstallationAdminSettings } from "@gitpod/gitpod-protocol";
+import { Data, InstallationAdminSettings } from "@gitpod/gitpod-protocol";
 import { AdminContext } from "../admin-context";
 import CheckBox from "../components/CheckBox";
 import { PageWithSubMenu } from "../components/PageWithSubMenu";
 import { getGitpodService } from "../service/service";
 import { adminMenu } from "./admin-menu";
+import { useEffect, useState } from "react";
+import InfoBox from "../components/InfoBox";
 
 export default function Settings() {
     const { adminSettings, setAdminSettings } = useContext(AdminContext);
+    const [telemetryData, setTelemetryData] = useState<Data>();
+
 
     const actuallySetTelemetryPrefs = async (value: InstallationAdminSettings) => {
         await getGitpodService().server.adminUpdateSettings(value);
         setAdminSettings(value);
     }
 
+    useEffect(() => {
+        (async () => {
+            const data = await getGitpodService().server.adminGetTelemetryData();
+            setTelemetryData(data)
+        })();
+    });
+
     return (
         <div>
             <PageWithSubMenu subMenu={adminMenu} title="Settings" subtitle="Configure settings for your Gitpod cluster.">
                 <h3>Usage Statistics</h3>
                 <CheckBox
                     title="Enable Service Ping"
-                    desc={<span>This is used to provide insights on how you use your cluster so we can provide a better overall experience. <a className="gp-link" href="https://www.gitpod.io/privacy">Read our Privacy Policy</a></span>}
+                    desc={<span>This is used to provide insights on how you use your Gitpod instance so we can provide a better overall experience. <a className="gp-link" href="https://www.gitpod.io/privacy">Read our Privacy Policy</a>. The following data is sent:</span>}
                     checked={adminSettings?.sendTelemetry ?? false}
                     onChange={(evt) => actuallySetTelemetryPrefs({
                         sendTelemetry: evt.target.checked,
                     })} />
-            </PageWithSubMenu>
+                <InfoBox>{JSON.stringify(telemetryData, null, 2)}</InfoBox>
```

Comment:
Haven't done a lot of UI yet, Hence a newbie question. What's the nicest way to display this? 😬

Reply:
As a first step, I would recommend using a `pre` block.
```suggestion
                <InfoBox><pre>{JSON.stringify(telemetryData, null, 2)}</pre></InfoBox>
```

| before      | after |
| ----------- | ----------- |
| ![image](https://user-images.githubusercontent.com/24960040/154474085-25443ad2-421e-4779-ba89-c59a11e693cc.png) | ![image](https://user-images.githubusercontent.com/24960040/154473946-2222ed13-cca5-40ea-8a09-9eaf747bd303.png) |

With this proposed change, in my opinion, it would be sufficient for a first increment (as an MVC/minimal viable change).

For a next iteration, we could think about a toggle that allows collapsing the box. For that, we should ask @gtsiolis for his always terrific input on UX design.

Reply:
Updated this to use the `pre` block, and it already looks way better!

Reply:
Hey everyone!

**issue**: Using the alert component[[1](https://github.com/gitpod-io/gitpod/blob/bb809461b56dcc13a74d21faa1ec943fb0b8639b/components/dashboard/src/components/AlertBox.tsx)][[2](https://github.com/gitpod-io/gitpod/blob/bb809461b56dcc13a74d21faa1ec943fb0b8639b/components/dashboard/src/components/InfoBox.tsx)], also known as`InfoBox`, feels a bit misleading. Also, including the payload expanded looks a bit verbose and will not scale well in the long run as we add more information encapsulated in the payload. Lastly although a minor issue, the help links pattern (_Learn more_, _Read our Privacy Policy_, etc) is used at the end of sentences without any following text.

**suggestion**: What do you think of using a modal to encapsulate this information to avoid injecting this kind of verbose information in the dashboard layout?

**thought**: We could also :a: drop the gray background and default to the white background as well as :b: re-use the cancel action button if that's easier to do in the spirit of MVC (minimum viable change) using the existing modal component[[1](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/components/Modal.tsx)]. Ideally, in the long run, this could use a markdown component which we could either show under a toggle element or within a modal like the proposal below. 💭 

| Button | Modal |
|-|-|
| <img width="1440" alt="Button" src="https://user-images.githubusercontent.com/120486/154547527-b2a70d03-0a89-49c7-8323-cb1536866186.png"> | <img width="1440" alt="Modal" src="https://user-images.githubusercontent.com/120486/154547598-6261df81-39d4-493e-92e4-374657a0bd57.png"> |

Thanks for the ping @corneliusludmann! 🏓

Reply:
Thanks for our input, @gtsiolis! I agree with your proposal. :+1: 

If you agree, I would like to move this part to a new issue and exclude these improvements from this PR for the following reasons:
- It is functionally working now and I think it's very important that we get this into the February release.
- For @Pothulapati, it would take some time to get familiar with the modal feature. Moving this into a follow-up PR would give him more time for this without jeopardizing the release (or someone else takes the new issue to implement this improvement—Tarun can decide).

Thus, my proposal would be:
1. Let's merge this pull request as is.
2. Move the UX proposals to a new issue and do the work in a follow-up PR.

Reply:
Took the liberty and created https://github.com/gitpod-io/gitpod/issues/8300.

Reply:
Thanks @corneliusludmann for opening the follow up issue. Agree! Re-posting from https://github.com/gitpod-io/gitpod/pull/8125#issuecomment-1036261804:

![153608231-bad67564-6781-477d-af76-a57d7cbc047a](https://user-images.githubusercontent.com/120486/154678363-9b1f6a6f-e81b-44e4-87eb-bdfcfe9e6653.jpg)